### ISSUE TITLE
Can't resolve 'react/lib/ReactMount'  issue

### ISSUE THREAD

**Issue by bairisuresh** [stylesuxx/generator-react-webpack-redux#55]
I have installed latest generator-rect-webpack-redux,. after running npm start it is throwing below error.
ERROR in ./src/reducers/index.js
Module not found: Error: Can't resolve 'react/lib/ReactMount' in 'C:\Users\folder11\Desktop\demo\src\reducers'
 @ ./src/reducers/index.js 1:307-338

this is repeated for all js files in scr folder...
Can anyonce help me out to resolve this plzz

**Comment by stylesuxx** [comment]
Can you Plesse post mode and npm Version?

Am 17.11.2016 10:07 schrieb "sureshbairi" <notifications@github.com>:

**Comment by bairisuresh** [comment]
@ stylesuxx     Node - 6.5.0
NPM- 3.10.8

**Comment by bairisuresh** [comment]
@stylesuxx
Node -6.5.0
NPM : 3.10.8

**Comment by stujo** [comment]
I think I was able to fix this with these changes:

putsch is the name of the app I made with the generator

```
--- a/client/putsch/.babelrc
+++ b/client/putsch/.babelrc
@@ -1,11 +1,12 @@
 {
   "presets": [
-    "es2015-native-modules",
+    ["es2015", { "modules": false }],
     "react",
     "airbnb"
   ],
   "plugins": [
     "transform-decorators-legacy",
-    "transform-object-rest-spread"
+    "transform-object-rest-spread",
+    "react-hot-loader/babel"
   ]
 }
```


```
--- a/client/putsch/conf/webpack/Base.js
+++ b/client/putsch/conf/webpack/Base.js
@@ -166,7 +166,7 @@ class WebpackBaseConfig {
               this.includedPackages,
               [this.srcPathAbsolute]
             ),
-            loaders: ['react-hot', 'babel']
+            loaders: ['babel']
           }
         ]
       },
```

```
--- a/client/putsch/package.json
+++ b/client/putsch/package.json
@@ -59,18 +59,18 @@
     "null-loader": "^0.1.1",
     "phantomjs-prebuilt": "^2.1.7",
     "react-addons-test-utils": "^15.0.1",
-    "react-hot-loader": "^1.3.0",
+    "react-hot-loader": "^3.0.0-beta",
     "rimraf": "^2.5.2",
     "sass-loader": "^3.2.0",
     "sinon": "^1.17.3",
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
-    "webpack": "^2.1.0-beta.6",
-    "webpack-dev-server": "^2.0.0-beta"
+    "webpack": "2.1.0-beta.22",
+    "webpack-dev-server": "2.1.0-beta.2"
   },
   "dependencies": {
-    "react": "^15.0.1",
-    "react-dom": "^15.0.1",
+    "react": "^15.4.0",
+    "react-dom": "^15.4.0",
     "react-redux": "^4.4.6",
     "redux": "^3.6.0"
   }
```

**Comment by stylesuxx** [comment]
Just checked, rect version is to new, please use 15.3.0 for now:

    npm install react@15.3.0

**Comment by bairisuresh** [comment]
Previously i have used react@15.0.1 in my project.
Now i have changed it to react@15.3.0 , but getting same error.

**Comment by bairisuresh** [comment]
Hi @stujo ,
It's working if i did the changes that you have mentioned above. but hot-loading is not working , i need to refresh the page every time i change my code.
is there any solution for this?

**Comment by creativecoder** [comment]
@bairisuresh These 2 steps worked for me:

1. Update react-hot-loader to `3.0.0-beta.6`
1. Remove `react-hot` from loaders in `conf/webpack/Base.js`

**Comment by stylesuxx** [comment]
This is a problem in the template of the base generator, we are working on it. Please follow @creativecoder 's recommendation.

**Comment by bairisuresh** [comment]
@stylesuxx ,
thank you i will continue with the workaround solution for now.

**Comment by stylesuxx** [comment]
Fixed in 1.1.0 - feel free to re-open if problem persists.
